pkg/networkmanager/network-interface-members.jsx

```diff
@@ -192,7 +192,7 @@ export const NetworkInterfaceMembers = ({
     const add_btn = (
         <Dropdown onSelect={() => setIsOpen(false)}
                   toggle={
-                      <DropdownToggle id="add-member" onToggle={setIsOpen}>
+                      <DropdownToggle id="add-member" onToggle={(_, isOpen) => setIsOpen(isOpen)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)